addons/account/models/account_move.py

```diff
@@ -1781,12 +1781,12 @@ def _move_autocomplete_invoice_lines_create(self, vals_list):
 
             is_invoice = vals.get('move_type') in self.get_invoice_types(include_receipts=True)
 
-            if 'line_ids' in vals:
+            if vals.get('line_ids'):
                 vals.pop('invoice_line_ids', None)
                 new_vals_list.append(vals)
                 continue
 
-            if is_invoice and 'invoice_line_ids' in vals:
+            if is_invoice and vals.get('invoice_line_ids'):
                 vals['line_ids'] = vals['invoice_line_ids']
```

Comment:
It doesn't make sense to have both `line_ids` and `invoice_line_ids` set in vals, but it make sense to have either to be empty. Using `get` instead of `in` would prevent that.

Reply:
I see, so the correct fix for this case would be to remove the  `line_ids` in the [repair vals](https://github.com/odoo/odoo/blob/14.0/addons/repair/models/repair.py#L343-L354), as it isn't being used?

I just tested this locally and it also fixes the issue

Reply:
Yes.

You can keep the PR, but you will need to change your commit message.

Reply:
@hildickethan You will also need to sign the Odoo CLA : https://github.com/odoo/odoo/blob/14.0/doc/cla/sign-cla.md

Reply:
My company has signed it in this PR https://github.com/odoo/odoo/pull/68244, I'm included in the contributor list